content/zh/docs/setup/additional-setup/getting-started/index.md

```diff
@@ -223,7 +223,7 @@ Bookinfo 应用已被部署但还不能从外部进行访问。
 
     {{< text bash >}}
     $ echo "$GATEWAY_URL"
-    169.48.8.37:80
+    127.0.0.1:80
```

Comment:
https://github.com/istio/istio.io/blob/master/content/en/docs/setup/additional-setup/getting-started/index.md?plain=1#L240
The English documentation here is also seems `169.48.8.37:80`